force-app/main/default/lwc/wireGetRecord/wireGetRecord.html

```diff
@@ -0,0 +1,17 @@
+<template>
+    <lightning-card title="WireGetRecord" icon-name="custom:custom67">
+        <div class="slds-var-m-horizontal_medium">
+            <template if:true={record.data}>
+                <div class="scroller">
+                    <pre>{recordStr}</pre>
+                </div>
+            </template>
+        </div>
+        <template if:true={record.error}>
+            <c-error-panel errors={record.error}></c-error-panel>
+        </template>
+        <c-view-source source="lwc/WireGetRecord" slot="footer">
+            Get Record
```

Comment:
Can you please provide more description, something like 'Get a record's data using @wire'. Also, change the description for other components.

Reply:
Hi @gravitypriest, Thank you for updating the PR with the new commits. I have left a couple of minor comments. Also, Can you please run prettier once and fix the formatting issues?